lasagne/layers/dense.py

```diff
@@ -111,11 +115,56 @@ def get_output_for(self, input, **kwargs):
 
 class NINLayer(Layer):
     """
-    Network-in-network layer.
+    Network-in-network layer [1]_.
     Like DenseLayer, but broadcasting across all trailing dimensions beyond the
     2nd.  This results in a convolution operation with filter size 1 on all
     trailing dimensions.  Any number of trailing dimensions is supported,
     so NINLayer can be used to implement 1D, 2D, 3D, ... convolutions.
+
+    Parameters
+    -----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape
+
+    num_units : int
+        The number of units of the layer
+
+    untie_biases : bool
+        If true a separate bias vector is used for dimension trailing the 2nd.
```

Comment:
You might want to copy the `untie_biases` description from one of the convolution layers as well, see .e.g. https://github.com/Lasagne/Lasagne/blob/master/lasagne/layers/conv.py#L132